services/gundeck/test/integration/API.hs

```diff
@@ -684,6 +685,27 @@ testReplaceSharedPushToken g b = do
         [t2] @=? ts1
         [t2] @=? ts2
 
+testLongPushToken :: Gundeck -> Brig -> Http ()
+testLongPushToken g b = do
+    uid <- randomUser b
+    clt <- randomClient g uid
+    
+    -- normal size APNS token should succeed
+    tkn1 <- randomApnsToken clt defTS
+    registerPushTokenRequest uid tkn1 g !!! const 201 === statusCode
+
+    -- APNS token over 400 bytes should fail (actual token sizes are twice the tSize)
+    tkn2 <- randomApnsToken clt defTS{tSize=256}
```

Comment:
`defTS{tSize=256}` seems an odd thing to do, since `defTS` is `TokenSpec 32 appName`, i.e. already specifies a length of 32, which you then override again. How about simply using `(TokenSpec 256 appName)` instead?

Reply:
I did it this way so that a default TokenSpec could be specified, and have arbitrary parameters overridden, instead of statically specifying the same values again for the same use cases.